spec/index.html

```diff
@@ -732,12 +754,16 @@ <h2>Literals</h2>
         not defined by this specification.</li>
     </ol>
 
-    <p><dfn data-local-lt="term-equal">Literal term equality</dfn>: Two literals are term-equal (the same
-      RDF literal) if and only if the two <a>lexical forms</a>,
-      the two <a>datatype IRIs</a>, and the two
-      <a>language tags</a> (if any) compare equal,
-      character by character. Thus, two literals can have the same value
-      without being the same RDF term. For example:</p>
+    <p><dfn data-local-lt="term-equal">Literal term equality</dfn>:
+      Two literals are term-equal (the same <a>RDF literal</a>)
+      if and only if the two <a>lexical forms</a>,
+      the two <a>datatype IRIs</a>,
+      the two <a>language tags</a> (if any) normalized to lower case, and
+      the two <a>base directions</a> (if any),
+      all compare equal, character by character.
+      Thus, two literals can have the same value
+      without being the same <a>RDF term</a>.
```

Comment:
This is the text containing the change in term-equality, cited by @afs

Reply:
There's an additional bit about potentially normalizing language tags based on BCP47 rules that can be moved to the new PR as well.

Reply:
See https://github.com/w3c/rdf-concepts/pull/48/commits/c45d9470785a9c681c1ad2cf8cf47906b4d62c75.